pkg/terra/contract_reader.go

```diff
@@ -190,7 +190,7 @@ func (r *OCR2Reader) LatestTransmissionDetails(ctx context.Context) (
 //		return
 //	}
 //	blockNum := uint64(latestBlock.Block.Header.Height) - uint64(lookback/cc.cfg.BlockRate())
-//	res, err := cc.chainReader.TxsEvents([]string{fmt.Sprintf("tx.height>=%d", blockNum+1), fmt.Sprintf("wasm-new_round.contract_address='%s'", cc.address.String())})
+//	res, err := cc.chainReader.TxsEvents([]string{fmt.Sprintf("tx.height>=%d", blockNum+1), fmt.Sprintf("wasm-new_round.contract_address='%s'", cc.address.String())}, nil)
```

Comment:
:scissors:

Reply:
I don't know why that code is commented and left in the source. I won't remove it because it's not part of this PR.